backend/internetarchive/internetarchive.go

```diff
@@ -151,6 +151,11 @@ Owner is able to add custom keys. Metadata feature grabs all the keys including
 			Help:     "Host of InternetArchive Frontend.\n\nLeave blank for default value.",
 			Default:  "https://archive.org",
 			Advanced: true,
+		}, {
+			Name:     "metadata",
+			Help:     "Metadata to be set on the item.\n\nFormat is key=value. Note: x-archive-meta- prefix is automatically added.",
```

Comment:
Can you emphasize that the metadata here is the item metadata and that is distinct from the file metadata which rclone uses with `--metadata` support.